docs/src/components/Footer/index.js

```diff
@@ -108,54 +110,62 @@ const LinksContactUs = () => {
-  useEffect(() => {
-    fetch(githubApiContributors)
-      .then((res) => res?.json())
-      .then(
-        (result) => {
-          setIsLoaded(true);
-          setItems(result);
-        },
-        (error) => {
-          setIsLoaded(true);
-          console.error(error);
-        }
-      );
-    return () => {
-      setItems([]);
-    };
-  }, []);
+const Contributors = ({ isTopContributors }) => {
   return (
     <div className="col">
-      {isLoaded && items.length && (
+      {(
         <>
-          <span className="footer__title">
-            <Translate
-              id="component.Footer.topContributors"
-              description="Top contributors label in footer"
-            >Top contributors</Translate>
-          </span>
-          <ul className="footer__items">
-            {items?.slice(0, 3).map((item) => {
-              return (
-                <li className="footer__item" key={item.login}>
-                  <Link
-                    className="footer__link-item"
-                    to={item.html_url}
-                    target="_blank"
-                  >
-                    {item.login}
-                  </Link>
-                </li>
-              );
-            })}
-          </ul>
+          {
+            isTopContributors ? (
+              <>
+                <span className="footer__title">
+                  <Translate
+                    id="component.Footer.topContributors"
+                    description="Top contributors label in footer"
+                  >Top contributors</Translate>
+                </span>
+                <ul className="footer__items">
+                  {topContributors?.slice(0, 6).map((item) => {
+                    return (
+                      <li className="footer__item" key={item}>
+                        <Link
+                          className="footer__link-item"
+                          to={githubProfile(item)}
+                          target="_blank"
+                        >
+                          {item}
+                        </Link>
+                      </li>
+                    );
+                  })}
+                </ul>
+              </>
+            ) : (
+              <>
+                <span className="footer__title">
+                  <Translate
+                    id="component.Footer.newContributors"
+                    description="New contributors label in footer"
+                  >New contributors</Translate>
+                </span>
+                <ul className="footer__items">
+                {newContributors?.slice(0, 6).map((item) => {
+                  return (
+                    <li className="footer__item" key={item}>
+                      <Link
+                        className="footer__link-item"
+                        to={githubProfile(item)}
+                        target="_blank"
+                      >
+                        {item}
```

Comment:
```suggestion
                        {githubProfile(item)}
```
we are only displaying the username in website

Reply:
ok